GrizzFit/src/screens/Register.js

```diff
@@ -0,0 +1,126 @@
+import { useNavigation } from '@react-navigation/native'
+import React, { useContext, useState } from 'react'
+import { KeyboardAvoidingView, StyleSheet, Text, View } from 'react-native'
+import { TextInput, TouchableOpacity } from 'react-native-gesture-handler'
+import { AuthContext } from '../navigation/AuthProvider'
+import { darkModePalette } from '../styles/DarkModePalette'
+
+const Register = () => {
+    const [userName, setName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+
+    const {register} = useContext(AuthContext)
+
+    const navigation = useNavigation()
+
+    const handleRegister = () => {
+        // Firebase Auth account creation
+        register(email, password);
+
+        // TODO: Create a new user document in Firestore and add their name to it
+    }
+
+    const navLogin = () => {
+        navigation.navigate("Login")
+    } 
+
+    return (
+        <KeyboardAvoidingView
```

Comment:
Not sure if we could get this KeyboardAvoidingView to work on Android too :/

Reply:
A quick glance at the docs seems to say that the `behavior` attribute should be set dynamically based on the OS detected. 

https://reactnative.dev/docs/keyboardavoidingview#example

Reply:
So this should look something like this:
```jsx
        <KeyboardAvoidingView
            style={styles.container}
            behavior={Platform.OS === "ios" ? "padding" : "height"}
        >
```
Just gotta import Platform.OS.